### PR TITLE
Extract connect icon to component

### PR DESCRIPTION
**Pull request by luiscantero**
* Extract connect status icon to component for better code reuse
* Simplify some razor code
* Add semicolon to razor usings as it makes it easier to move code to code-behind
* Fix some formatting inconsistencies

### CODE REVIEW

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerDiscoverer.razor [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿@*------------------------------------------------------------
-     Copyright (c) Microsoft Corporation.  All rights reserved.
-     Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
-    ------------------------------------------------------------*@
+  Copyright (c) Microsoft Corporation.  All rights reserved.
+  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+------------------------------------------------------------*@
 
 @using Microsoft.Azure.IIoT.App.Services
```
Comment (dacolgit) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerDiscoverer.razor:6]: add semicolon also here

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerPublisher.razor [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿@*------------------------------------------------------------
-     Copyright (c) Microsoft Corporation.  All rights reserved.
-     Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
-    ------------------------------------------------------------*@
+  Copyright (c) Microsoft Corporation.  All rights reserved.
+  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+------------------------------------------------------------*@
 
 @using Microsoft.Azure.IIoT.App.Common
 @using Microsoft.Azure.IIoT.App.Services
```
Comment (dacolgit) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerPublisher.razor:7]: add semicolon

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerPublisherContent.razor [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿@*------------------------------------------------------------
-    Copyright (c) Microsoft Corporation.  All rights reserved.
-    Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+  Copyright (c) Microsoft Corporation.  All rights reserved.
+  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
 ------------------------------------------------------------*@
 
 @using Microsoft.Azure.IIoT.App.Services
```
Comment (dacolgit) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerPublisherContent.razor:6]: add semicolon

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerPublisher.razor (hunk 2 above)
Comment (luiscantero) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerPublisher.razor:7]: Done

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerDiscoverer.razor (hunk 1 above)
Comment (luiscantero) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerDiscoverer.razor:6]: Done

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerPublisherContent.razor (hunk 3 above)
Comment (luiscantero) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerPublisherContent.razor:6]: Done

**Pull request merged**